### PR TITLE
qFlipper: 1.0.1 -> 1.0.2

### PR DESCRIPTION
**Pull request by cab404**
Also checked and enabled updateScript

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/qflipper/default.nix [hunk 1]
```diff
@@ -19,12 +19,14 @@
...
+, nix-update-script
 }:
 let
-  version = "1.0.1";
+  pname = "qFlipper";
+  version = "1.0.2";
   timestamp = "99999999999";
   commit = "nix-${version}";
-  hash = "sha256-vHBlrtQ06kjjXXGL/jSdpAPHgqb7Vn1c6jXZVXwxHPQ=";
+  sha256 = "vHBlrtQ06kjjXXGL/jSdpAPHgqb7Vn1c6jXZVXwxHPQ=";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/qflipper/default.nix:29]: Please revert that. sri hashes should have a prefix.

Comment (cab404) [reply pkgs/tools/misc/qflipper/default.nix:29]: @SuperSandro2000 updaterScript could not process SRI w/o hash — and I thought that if you specify hash type, you can waiver type from the hash itself  :\

Comment (cab404) [reply pkgs/tools/misc/qflipper/default.nix:29]: Oh wait, I guess it did not, the hash is the same.

Comment (cab404) [reply pkgs/tools/misc/qflipper/default.nix:29]: Ok, tautology did work, but it's not pretty :(

**Comment by gador** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 170951` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>qFlipper</li>
</ul>
</details>

**Review by gador — APPROVED** [review]

File: pkgs/tools/misc/qflipper/default.nix [hunk 2]
```diff
@@ -19,12 +19,14 @@
...
 , qtgraphicaleffects
 , qtwayland
+, nix-update-script
 }:
 let
-  version = "1.0.1";
+  pname = "qFlipper";
+  version = "1.0.2";
+  sha256 = "sha256-CJQOEUwYPNd4x+uBINrxeYVITtYrsEFaYLHQh2l12kA=";
   timestamp = "99999999999";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/qflipper/default.nix:28]: Why is this a variable and not just hardcoded in the scrip?

File: pkgs/tools/misc/qflipper/default.nix [hunk 3]
```diff
@@ -19,12 +19,14 @@
...
   commit = "nix-${version}";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/qflipper/default.nix:29]: Same for this.

**Pull request merged**